server/src/test/java/io/druid/server/ClientInfoResourceTest.java

```diff
@@ -83,10 +126,10 @@ public void setup()
   }
 
   @Test
-  public void testGetDatasourceNonFullWithLargeInterval()
+  public void testGetDatasourceNonFullWithInterval()
   {
-    Map<String, Object> actual = resource.getDatasource(dataSource, "1975/2050", null);
-    Map<String, ?> expected = ImmutableMap.of(
+    Map<String, Object> actual = resource.getDatasource(dataSource, "1975/2015", null);
```

Comment:
yeah.. Because I added some new segments in that interval, and I still wanted to reuse this test case, so I shortened the interval. I added testGetDatasourceNonFullWithLargeInterval() in the end so that 1975/2050 is still tested.